src/lib.rs

```diff
@@ -27,25 +27,33 @@
 
 #![no_std]
 
+extern crate byteorder;
+extern crate digest;
+
 #[cfg(test)]
 extern crate std;
 
 #[cfg(feature = "std")]
 extern crate rand;
 
-#[cfg(feature="serialize")]
+#[cfg(feature = "serialize")]
 extern crate serde;
-#[cfg(feature="serialize")]
-use serde::{Serialize, Deserialize};
+#[cfg(feature = "serialize")]
+use serde::{Deserialize, Serialize};
 
 mod number_streams;
 mod thirty_two;
 
-pub use thirty_two::XxHash as XxHash32;
 #[cfg(feature = "std")]
 pub use thirty_two::RandomXxHashBuilder as RandomXxHashBuilder32;
+pub use thirty_two::XxHash as XxHash32;
 
+use byteorder::{BigEndian, ByteOrder};
 use core::hash::Hasher;
+use digest::{
```

Comment:
You may want to use a similar technique as we do for [libstd support](https://github.com/shepmaster/twox-hash/blob/master/src/std_support.rs) and create a `digest_support.rs` file. That way, the conditional compilation can be restricted to a small number of places (I think just the `extern crate`s and that `use digest_support`).

Reply:
That makes sense, I'll do it now :)